src/devices/KeyMatrix/samples/KeyMatrix.Sample.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Device.Gpio;
+using System.Linq;
+using System.Threading;
+
+namespace Iot.Device.KeyMatrix.Samples
+{
+    /// <summary>
+    /// This sample is for Raspberry Pi Model 3B+
+    /// </summary>
+    internal class Program
+    {
+        public static void Main(string[] args)
+        {
+            // get arguments
+            Console.Write("input output pins(eg. 26,19,13,6) ");
+            var line = Console.ReadLine();
+            IEnumerable<int> outputs = (string.IsNullOrEmpty(line) ? "26,19,13,6" : line).Split(',').Select(m => int.Parse(m));
+            Console.Write("input input pins(eg. 21,20,16,12) ");
+            line = Console.ReadLine();
+            IEnumerable<int> inputs = (string.IsNullOrEmpty(line) ? "21,20,16,12" : line).Split(',').Select(m => int.Parse(m));
+            Console.Write("input scaning interval(eg. 15) ");
+            line = Console.ReadLine();
+            int interval = int.TryParse(line, out int i) ? i : 15;
+            Console.Write("input read key event count(eg. 20) ");
+            line = Console.ReadLine();
+            int count = int.TryParse(line, out int c) ? c : 20;
+
+            // get GPIO controller
+            GpioController gpio = new GpioController();
+
+            // you can also use other GPIO controller
+            /*
```

Comment:
I don't think this is that useful here, you can put it as a sample on the README page though